core/domain/beam_job_services_test.py

```diff
@@ -82,8 +97,9 @@ def assert_domains_equal_models(self, beam_job_runs, beam_job_run_models):
         """
         self.assertEqual(len(beam_job_runs), len(beam_job_run_models))
         runs = sorted(beam_job_runs, key=lambda j: j.job_id)
-        models = sorted(beam_job_run_models, key=lambda m: m.id)
-        for i, (r, m) in enumerate(python_utils.ZIP(runs, models)):
+        # NOTE: MyPy is wrong, ID is a string.
+        run_models = sorted(beam_job_run_models, key=lambda m: m.id) # type: ignore[no-any-return]
```

Comment:
This type ignore can be avoided. This can be done by assigning the lambda function to a separate variable as shown [here](https://github.com/oppia/oppia/blob/63d453f703406e32f47b3706ae900bfc346a67ae/core/storage/question/gae_models_test.py#L718).

Reply:
Done.